### DISCUSSION TITLE
Issue with cirrus-ci

### DISCUSSION THREAD

**Discussion by ThakurRahulSingh** [ROM-builders/temporary#14423] [Q&A]
Idk what is this error but many people facing this error in PixelExperience thirteen but no one reported yet.

1. https://cirrus-ci.com/build/6728007078379520
2. https://cirrus-ci.com/build/5208305245618176
3. https://cirrus-ci.com/build/5794861478576128
4. https://cirrus-ci.com/build/5365608555479040

@Apon77 bro please check this issue

**Comment by Apon77** [comment]
Please don't mention me all the time. I always see when there is a discussion in github.

**Reply by ThakurRahulSingh** [reply]
I am Sorry.
